src/components/PaymentMethodForm/PaymentMethodForm.scss

```diff
@@ -23,6 +23,17 @@
             & .deriv-input__container {
                 width: 100%;
             }
+
+            & .payment-method-form__fields {
+                & .payment-method-form__field-wrapper {
+                    & .deriv-input__container {
+                        & .deriv-input__right-content {
+                            display: flex;
+                            align-items: center;
+                        }
+                    }
+                }
+            }
```

Comment:
is it possible to just target the .deriv-input__right-content inside of the field wrapper below on line 130